src/config/section/role.js

```diff
@@ -35,12 +35,16 @@ export default {
       icon: 'plus',
       label: 'label.add.role',
       listView: true,
-      args: ['name', 'description', 'type'],
-      mapping: {
-        type: {
-          options: ['Admin', 'DomainAdmin', 'User']
-        }
-      }
+      popup: true,
+      component: () => import('@/views/iam/CreateRole.vue')
+    },
+    {
+      api: 'importRole',
+      icon: 'cloud-upload',
+      label: 'Import Role',
```

Comment:
@sureshanaparti please move this to locales/en.json; add a new key.

Reply:
@rhtyd Updated with new key in _locales/en.json_